recipes/menuinst/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "1.4.8" %}
+{% set sha256 = "053994cea32d71acf2846efc6dbd97b2fe5a4e98fa105ac36098d0a6d9c4ad9e" %}
+
+package:
+  name: menuinst
+  version: {{ version }}
+
+source:
+  fn: menuinst-{{ version }}.tar.gz
+  url: https://github.com/ContinuumIO/menuinst/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script:
+    - python setup.py install
+    - copy "%SRC_DIR%\\cwp.py" "%PREFIX%\\"
+  entry_points:
+    - menuinst = menuinst.main:main
+  skip_compile_pyc:
+    - cwp.py
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - pywin32
+
+test:
+  imports:
+    - menuinst
+
+  commands:
+    - menuinst --help
+
+about:
+  home: https://github.com/ContinuumIO/menuinst
+  license: BSD 3-Clause
```

Comment:
Should this have a license_family?

Reply:
It could. Skipped in the interest of making sure it built correctly first. Happy to add now or later.

Reply:
Done below.